ex1/merge_utils.py

```diff
@@ -1,19 +1,32 @@
-def merge_dicts(source, dest):
+import copy
+
+
+def merge_dicts(source: dict, dest: dict):
```

Comment:
source and dest implies that the source's keys are merged into dest's keys. I named it like that so that the smaller object would usually be the source, and the big object the destination, so that the program will run faster